tox.ini

```diff
@@ -20,19 +20,17 @@ deps =
     flake8
     Pillow
     mock
-; TODO: This will ONLY work on linux systems.  If we could get this from PyPI,
-;       then the platform resolution could be handled by pip instead of here.
-    py27: https://github.com/markreidvfx/pyaaf/releases/download/v1.0.0/PyAAF-1.0.0-cp27-cp27mu-manylinux1_x86_64.whl
-    py35: https://github.com/markreidvfx/pyaaf/releases/download/v1.0.0/PyAAF-1.0.0-cp35-cp35m-manylinux1_x86_64.whl
-    py36: https://github.com/markreidvfx/pyaaf/releases/download/v1.0.0/PyAAF-1.0.0-cp36-cp36m-manylinux1_x86_64.whl
 
 commands =
+    git clone https://github.com/markreidvfx/pyaaf2
```

Comment:
Unless we're trying to get a specific branch here, we should rely on the new line 26 below to pull pyaaf2 from [pypi](https://pypi.org/project/pyaaf2/). Right now this git clone line will float as new commits come in on master, which is probably not desirable for testing.